client/src/api-client.ts

```diff
@@ -145,7 +167,7 @@ class FetchApiClient implements ApiClient {
 
   async deleteItem(itemId: number): Promise<void> {
     const response = await fetch(
-      `${import.meta.env.VITE_SERVER_URL}/items/${itemId}`,
+      `${import.meta.env.VITE_SERVER_URL}/review-items/${itemId}`,
```

Comment:
Thanks for applying the URI modification.

In the logic of server controllers in Spring Boot, I centralize shared URI segments (e.g. `/review-items`) as a shared prefix for all endpoint paths within the controller, making it easier to maintain and organize related routes.

What do you think about managing the common URI segment as a constant?

Reply:
Good idea! I didn’t think it was time to reduce redundant things yet, but after hearing your suggestion, I agree—it’s time. I’ll incorporate your feedback in the next PR!

Reply:
Thanks @sounmind !